addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -41,12 +48,32 @@ export class ProjectTaskFormController extends FormController {
         const versionedFieldName = 'description';
         const historyMetadata = record.data["html_field_history_metadata"]?.[versionedFieldName];
         if (!historyMetadata) {
+            this.notifications.add(
+                markup(
+                    `<span>${escape(
```

Comment:
Do we need the span? It doesn't seem to do anything

Reply:
Though it was necessary to put the escaped string into some "balises" of any kind, but ok if it not needed.